schema@v1.1.1/row-meta-schema.json

```diff
@@ -146,9 +146,23 @@
     "additionalRelations": {
       "type": "object"
     },
-    "isTemporal": {
-      "type": "boolean"
-    },
+    "temporal": {
```

Comment:
Dit veld is niet verplicht. Ik zou hier verwachten dat dat op een of andere manier in het schema wordt uitgedrukt. Of gaat dit in een keer goed?

Reply:
Volgens mij zijn properties in json schemas standaard toegestaan maar niet verplicht. Daarvoor moeten ze aan de required array worden toegevoegd

Reply:
Ah idd, je heb gelijk. Ik heb-em nu ook even uitgechecked en de validatie doet het al een zonnetje :-)